jstl/src/main/resources/com/sun/ts/tests/jstl/spec/fmt/format/parsedate/positivePDFallbackLocaleTestJava20Plus.gf

```diff
@@ -11,13 +11,14 @@
 
     <!-- The fallbackLocale configuration variable will be
              used if no locale match can be determined. -->
-
-
-
+    
+    
+    
     Nov 21, 2000<br>
     3:45:02 AM<br>
-    Nov 21, 2000, 3:45:02 AM<br>
+    Nov 21, 2000, 3:45:02 AM<br>
```

Comment:
@pnicolucci did you intend to update line 19 of jstl/src/main/resources/com/sun/ts/tests/jstl/spec/fmt/format/parsedate/positivePDFallbackLocaleTestJava20Plus.gf?

Reply:
Yes, I did as it was a standard space and not a NNBSP as it should have been for Java 20+